gammapy/data/observations.py

```diff
@@ -722,7 +727,7 @@ def index(self, key):
 
     @property
     def ids(self):
-        """List of obs IDs (`list`)"""
+        """List of obs IDs (`list`)."""
```

Comment:
```suggestion
        """List of observation IDs (`list`)."""
```

Reply:
What about the `(list)` here ? I think it is ok but it could also simply get removed.